pages/tutorials/getting_started.md.erb

```diff
@@ -8,33 +8,39 @@ This guide will help you setup your first Buildkite pipeline. If you don’t hav
+1. Follow the [Agent installation instructions](/docs/agent/v3/installation) to install and start an agent.
+2. Make sure you've configured the agent token, as that is what connects the agent to your Buildkite account. Refer to [Buildkite Agent Configuration](/docs/agent/v3/configuration) for more information.
+3. Opt in to the YAML steps editor. This is the newer way of configuring pipelines. Refer to [Pipeline Upgrade - Using YAML Steps for new pipelines](/docs/tutorials/pipeline-upgrade#using-yaml-steps-for-new-pipelines) for more information.
 
 ## Add the sample pipeline
 
-[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that will be run on your code.
-
-We’ve created a [bash-example test repository](https://github.com/buildkite/bash-example) to use as your first pipeline. Click the “Add to Buildkite” button below to open a new pipeline page for the example pipeline:
-
-<a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
+[Pipelines](/docs/pipelines/) are how Buildkite represents your CI workflow, a series of steps that you run on your code.
 
-Accept the pre-filled defaults, and click *Create Pipeline*.
+We've created sample pipelines to use as your first pipeline:
 
-## Run your first build
+1. Click the appropriate *Add to Buildkite* button:
 
-Now that you’re running an agent and have created a pipeline you’re ready to run your first build. Click the *New Build* button in the top right corner:
+    [bash-example test repository](https://github.com/buildkite/bash-example) 
 
-<%= image "new-build.png", size: "270x60", alt: "New build button" %>
+    <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/bash-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add Bash Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
-Type in a build message (being sure to add some [emojis](/docs/pipelines/emojis)) and click *Create Build*:
+    [powershell-example test repository](https://github.com/buildkite/powershell-example)
 
-<%= image "trigger-build.png", size: "600x460", alt: "New build dialog" %>
+    <a class="inline-block" href="https://buildkite.com/new?template=https://github.com/buildkite/powershell-example" target="_blank" rel="nofollow"><img src="https://buildkite.com/button.svg" alt="Add PowerShell Example to Buildkite" class="no-decoration" width="160" height="30"></a>
 
-The build will run, and once it’s finished it’ll turn green!
+2. Accept the pre-filled defaults, and select *Create Pipeline*.
+3. The example repositories contain a `pipeline.yml` setting out the pipeline steps, so on the *Steps* screen, add the following to upload and use the `pipeline.yml`:
```

Comment:
I have mixed feelings about this switch. It is definitely an improvment, but it is gonna break if folks *don't* do the YAML pipelines step.

Reply:
Yes. However I've tried to signal clearly with the note, and I really don't think we should be using the deprecated method. Having several ways of handling pipeline config is messy, and having the default for new accounts be the less-favoured way is not ideal, but hopefully the docs somewhat address that if they steer people to the new way from the get-go.